src/consensus/pbft/pbft.h

```diff
@@ -4,6 +4,7 @@
 
 #include "consensus/ledgerenclave.h"
```

Comment:
Once the `global_commit_seqno` is updated from the global commit callback, we need to change `replicate()` to just return true.